src/binaryapi/TradingTimes.js

```diff
@@ -28,7 +28,7 @@ class TradingTimes {
                 let nextUpdate = this._nextUpdateDate();
 
                 if (!nextUpdate) {
-                    const now = new Date();
+                    const now = this._serverTime.getLocalDate();
```

Comment:
Try and see if you can change it to only use `getUTCDate`. With this you should only need `get`.